lib/qtcamconfig.cpp

```diff
@@ -118,9 +118,9 @@ QtCamConfig::QtCamConfig(QObject *parent) :
   d_ptr(new QtCamConfigPrivate) {
 
 #ifdef QT4
-  d_ptr->model = QtMobility::QSystemDeviceInfo().model().toLower();
+  d_ptr->model = QtMobility::QSystemDeviceInfo().productName().toLower();
```

Comment:
model: N9
productName: RM-696
Even if N9/N950 are without spaces, let's switch to model() for consistency (see JP-1301 as example).